cico_functions.sh

```diff
@@ -33,6 +33,30 @@ function load_jenkins_vars() {
   fi
 }
 
+function check_version() {
+  local query=$1
+  local target=$2
+  echo "$target" "$query" | tr ' ' '\n' | sort -V | head -n1 2> /dev/null
+}
+
+function check_buildx_support() {
```

Comment:
could you please add docs to this function?

Reply:
Added a comment to explain the functionality.